draft-ietf-ppm-dap.md

```diff
@@ -828,18 +828,18 @@ this cached lifetime with the Cache-Control header, as follows:
   Cache-Control: max-age=86400
 ~~~
 
-Clients SHOULD follow the usual HTTP caching {{!RFC9111}} semantics for key
+Clients SHOULD follow the usual HTTP caching {{!RFC9111}} semantics for HPKE
```

Comment:
Reviewer note: At some point in the past we called this the "key configuration" and didn't pick up the change here.